src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -318,6 +318,7 @@ private ValueTask TryWritePingAsync()
                 return default;
             }
 
+            // TODO: cancel?
```

Comment:
@BrennanConroy anything to be concerned about here?

Reply:
Haha, not really